PeerCo/small_scale_self_consumed_solar.md

```diff
@@ -0,0 +1,289 @@
+<!--
+
+This is a template to be used for methodology development. The content of this file shall follow the Markdown conventions with Github flavour.
+
+
+If you need help on the Markdown syntax for Github, here is a good link:
+https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax
+
+And if you need a tool to make the rendering work in VSCode, here is a good plugin:
+https://marketplace.visualstudio.com/items?itemName=bierner.markdown-preview-github-styles 
+
+--> 
+# Small scale solar self-consumption methodology
+
+**Name**: *Small scale solar self-consumption*
+
+**Developed by**: *[PeerCo](www.peerco.earth)*
+
+**Revision date**: *July 5th 2024*
+
+**Sectoral scope**:  *Energy / Energy Distribution / Energy Demand*
+
+
+## Summary description
+
+This methodology for measurements and verification of self-consumed small-scale on-site solar generation on a given site. The purpose of this methodology is to generate *Environmental Attribute Certificates* representing the self-consumed solar energy (at a specific time and geographical location), which can then be used in reporting or traded.
+
+## Definitions
+
+This section provides definitions of the key aspects of this methodology.
+
+### Normative language
+
+The following terminology is used throughout this document to outline requirements and guidelines (accordingly with common standardisation practices):
+- **shall** indicates a *requirement*,
+- **should** indicates a *recommendation*,
+- **may** indicates an *option* that is permitted.
+
+### Acronyms
```

Comment:
Comment for OpenEAC committee rather than on this specific method: should there be a central library of acronyms & glossary, to avoid a future situation where we are seeing the same terms used with different definitions?

Reply:
good idea

Reply:
In principle, that's a good idea. However, for validation purposes given that methodologies with a given version are considered *normative*, there would be a need to always synchronise a given methodology version with the corresponding glossary/acronym list etc. 
The current approach where methodologies are self-contained allows us to circumvent that complexity.